components/Button/button.css

```diff
@@ -0,0 +1,123 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif;
+} */
+
+:root {
+  --primary-color: #ff6d6d;
+  --disabled-bg-color: #f88e8e;
+  --secondary-color: rgb(213, 213, 213);
+  --secondary-txt-color: rgb(80, 80, 80);
+}
+
+.button_main_container {
```

Comment:
I would suggest you to give this a different class name so that in future, you don't get confused with another class that has the same name but spelt different.

Reply:
okay.